### ISSUE TITLE
Name property of a focusable element must not be null (Calendar date picker)

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#387]
**Repro Steps**
* Launch Calculator app
* Navigate to Menu and Select Date calculation
* Select Add or subtract days from the combo box
* Observe the issue using inspect.exe (available in C:\Program Files (x86)\Windows Kits\10\bin\<SDK Version>\x64)

**Actual Result**
Name Property is not defined for "From" Calendar picker.

**Expected Result**
Name Property should be defined for "From" Calendar picker. We should be leveraging the control header property instead of drawing our own label.

**Test Environment**
OS Version: Windows 10 Build 18351.1
App Name: Calculator 
App Version: 10.1902.16.0

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.

**Comment by dkheri** [comment]
Hi 
Am interested and working on this bug

**Comment by grochocki** [comment]
Thanks, @dkheri, it is all yours! Let us know if you have any questions. For more information on making changes to the code, check out our [contributing guidelines](https://github.com/microsoft/calculator/blob/master/CONTRIBUTING.md#git-workflow).
